src/DependencyInjection/SonataFormatterExtension.php

```diff
@@ -41,7 +41,7 @@ public function load(array $configs, ContainerBuilder $container)
 
         $bundles = $container->getParameter('kernel.bundles');
 
-        if (isset($bundles['IvoryCKEditorBundle'])) {
+        if (isset($bundles['FOSCKEditorBundle'])) {
```

Comment:
Please check both bundles, and maybe throw if both are registered.

Reply:
Please trigger a deprecation iff Ivory is registered

Reply:
So if they install fos on their own, we should use fos config else we continue doing it like now but throw deprecation

Reply:
and on next major we will replace ivory with fos